website/public/js/services/userServices.js

```diff
@@ -201,6 +198,16 @@ angular.module('habitrpg')
           })
         },
 
+        clearNewMessages: function () {
+          $http({
+            method: "POST",
+            url: 'api/v3/user/mark-pms-read',
+          })
+          .then(function (response) {
+            sync();
```

Comment:
Yea, I think the add hourglass and 10 gems should be the same, but I was trying to stray away from adding to much. I can definitely do it though.